src/infrastructure/blockchain/ethereum/EthereumOracleAdapter.ts

```diff
@@ -87,7 +87,7 @@ class EthereumOracleAdapter implements OracleGateway {
     try {
       await method.send({
         from: process.env.PUBLIC_KEY,
-        gas: await this.web3.eth.estimateGas({to: this.contract.address}),
+        gas: await method.estimateGas({from: process.env.PUBLIC_KEY}),
```

Comment:
I see, that new web3 repair "method.estimateGas" 👍

Reply:
`node_modules/web3-core/types/index.d.ts:31:23 - error TS2307: Cannot find module 'bignumber.js'.
31 import BigNumber from 'bignumber.js';
                         ~~~~~~~~~~~~~~
node_modules/web3-eth/types/index.d.ts:51:25 - error TS2307: Cannot find module 'bignumber.js'.
51 import {BigNumber} from 'bignumber.js';
                           ~~~~~~~~~~~~~~`

CircleCi has problem with bignumber from web3, please repair.